src/commands/basemaps-github/make.cog.github.ts

```diff
@@ -1,41 +1,51 @@
 import { DefaultColorRampOutput, DefaultTerrainRgbOutput, standardizeLayerName } from '@basemaps/config';
-import {
+import type {
   ConfigLayer,
   ConfigTileSet,
   ConfigTileSetRaster,
   ConfigTileSetVector,
-  TileSetType,
 } from '@basemaps/config/build/config/tile.set.js';
+import { TileSetType } from '@basemaps/config/build/config/tile.set.js';
 import { fsa } from '@chunkd/fs';
 
-import { logger } from '../../log.js';
-import { DEFAULT_PRETTIER_FORMAT } from '../../utils/config.js';
-import { GithubApi } from '../../utils/github.js';
-import { prettyPrint } from '../pretty-print/pretty.print.js';
+import { logger } from '../../log.ts';
+import { DEFAULT_PRETTIER_FORMAT } from '../../utils/config.ts';
+import { GithubApi } from '../../utils/github.ts';
+import { prettyPrint } from '../pretty-print/pretty.print.ts';
 
-export enum Category {
-  Urban = 'Urban Aerial Photos',
-  Rural = 'Rural Aerial Photos',
-  Satellite = 'Satellite Imagery',
-  Elevation = 'Elevation',
-  Event = 'Event',
-  Scanned = 'Scanned Aerial Imagery',
-  Other = 'New Aerial Photos',
-}
+export const Categories = {
+  Urban: 'Urban Aerial Photos',
+  Rural: 'Rural Aerial Photos',
+  Satellite: 'Satellite Imagery',
+  Elevation: 'Elevation',
+  Event: 'Event',
+  Scanned: 'Scanned Aerial Imagery',
+  Other: 'New Aerial Photos',
+} as const;
+
+export type Category = (typeof Categories)[keyof typeof Categories];
+
+// | 'Urban Aerial Photos'
+// | 'Rural Aerial Photos'
+// | 'Satellite Imagery'
+// | 'Elevation'
+// | 'Event'
+// | 'Scanned Aerial Imagery'
+// | 'New Aerial Photos';
 
 export interface CategorySetting {
   minZoom?: number;
   maxZoom?: number;
 }
 
 export const DefaultCategorySetting: Record<Category, CategorySetting> = {
-  [Category.Urban]: { minZoom: 14 },
-  [Category.Rural]: { minZoom: 13 },
-  [Category.Satellite]: { minZoom: 5 },
-  [Category.Elevation]: { minZoom: 9 },
-  [Category.Scanned]: { minZoom: 0, maxZoom: 32 },
-  [Category.Other]: {},
-  [Category.Event]: {},
+  'Urban Aerial Photos': { minZoom: 14 },
```

Comment:
```suggestion
  ['Urban Aerial Photos']: { minZoom: 14 },
```
Or why we can't do `Categories.Urban` now?

Reply:
I went back and foward on a couple options here, maybe Categories.Urban is better ? but then Categories.Other is "New Aerial Photos" so the key value pairs are somewhat weirdly defined ?


Maybe I should make the categories more consistent with their actual values, but then I could just use the strings?

Reply:
Can We add the missing [] as well.

Reply:
the `[]` were not needed so I removed them.